chainio/constructor/constructor.go

```diff
@@ -115,11 +112,13 @@ func BuildClients(config Config, logger logging.Logger) (*Clients, error) {
 
 }
 
-func (config *Config) buildElContractsChainClient(
-	logger logging.Logger,
+func (config *Config) buildElClients(
```

Comment:
Can we export this method? - eigenlayer-cli just cares about this and doesn't care about avs registry stuff. so it would be better to use this method.

Reply:
Yes absolutely we need to clean this constructor file to have smaller chunk initializations. Will be useful also for egnaddrs. Need to think how to organize all of this including building the bindings in bindings.go (could be moved here as a helper method)

Reply:
Added EL constructors, for eg https://github.com/Layr-Labs/eigensdk-go/pull/73/commits/3081f8a420ab2cb4608c875b187ff4977918a2e7#diff-78f2b53998897f8fe40be2a3043dec1f342d78f98c9ef5b5abf867a8160fe938R102